code/mission/missionparse.cpp

```diff
@@ -338,6 +338,70 @@ flag_def_list_new<Mission::Parse_Object_Flags> Parse_object_flags[] = {
+    { Mission::Parse_Object_Flags::SF_Ignore_count,					"Ignore this ship when counting ship types for goals."},
+    { Mission::Parse_Object_Flags::OF_Protected,					"Ship and Turret AI will ignore and not attack ship."},
+    { Mission::Parse_Object_Flags::SF_Reinforcement,				"This ship is a reinforcement ship."},
+    { Mission::Parse_Object_Flags::OF_No_shields,					"Ship will have no shields (ETS will be rebalanced if shields were off and are enabled)."},
+    { Mission::Parse_Object_Flags::SF_Escort,						"This ship is an escort ship."},
+    { Mission::Parse_Object_Flags::OF_Player_start,					"Ship is a player ship."},
+    { Mission::Parse_Object_Flags::SF_No_arrival_music,				"Don't play arrival music when ship arrives."},
+    { Mission::Parse_Object_Flags::SF_No_arrival_warp,				"No arrival warp in effect."},
+    { Mission::Parse_Object_Flags::SF_No_departure_warp,			"No departure warp in effect."},
+    { Mission::Parse_Object_Flags::SF_Locked,						"Ship cannot be changed in the Ship Loadout."},
+    { Mission::Parse_Object_Flags::OF_Invulnerable,					"Stops ship from taking any damage."},
+    { Mission::Parse_Object_Flags::SF_Hidden_from_sensors,			"If set, the ship can't be targeted and appears on radar as a blinking dot."},
+    { Mission::Parse_Object_Flags::SF_Scannable,					"Whether or not the ship can be scanned."},
+    { Mission::Parse_Object_Flags::AIF_Kamikaze,					"Ship will attack big ships by colliding with them and exploding."},
+    { Mission::Parse_Object_Flags::AIF_No_dynamic,					"Will stop allowing the AI to persue dynamic goals (eg: chasing ships it was not ordered to)."},
+    { Mission::Parse_Object_Flags::SF_Red_alert_store_status,		"Ship status should be stored/restored if red alert mission."},
+    { Mission::Parse_Object_Flags::OF_Beam_protected,				"Turrets with beam weapons will ignore and not attack ship."},
+    { Mission::Parse_Object_Flags::OF_Flak_protected,				"Turrets with flak weapons will ignore and not attack ship."},
+    { Mission::Parse_Object_Flags::OF_Laser_protected,				"Turrets with laser weapons will ignore and not attack ship."},
+    { Mission::Parse_Object_Flags::OF_Missile_protected,			"Turrets with missile weapons will ignore and not attack ship."},
+    { Mission::Parse_Object_Flags::SF_Guardian,						"Ship health cannot be reduced below 1."},
+    { Mission::Parse_Object_Flags::Knossos_warp_in,					"Ship uses the special Knossos warp in animation."},
+    { Mission::Parse_Object_Flags::SF_Vaporize,						"Causes a ship to vanish (no deathroll, no debris, no explosion) when destroyed."},
+    { Mission::Parse_Object_Flags::SF_Stealth,						"If set, the ship can't be targeted, is invisible on radar, and is ignored by AI unless firing."},
+    { Mission::Parse_Object_Flags::SF_Friendly_stealth_invis,		"If set, the ship can't be targeted even by ships on the same team."},
```

Comment:
'...if it is *also* flagged as stealth.'

Reply:
fixed